ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/carousel/CarouselComponentView.kt

```diff
@@ -243,12 +255,28 @@ private fun Indicator(
         progress,
     )
 
+    val shouldApplyStroke = (pageControl.default.strokeColor != null || pageControl.active.strokeColor != null) &&
+        (pageControl.default.strokeWidth != null || pageControl.active.strokeWidth != null)
+
+    val strokeColor = lerp(
+        (pageControl.default.strokeColor?.forCurrentTheme as? ColorStyle.Solid)?.color ?: Color.Transparent,
+        (pageControl.active.strokeColor?.forCurrentTheme as? ColorStyle.Solid)?.color ?: Color.Transparent,
+        progress,
+    )
+
+    val strokeWidth by animateDpAsState(
+        targetValue = targetStrokeWidth,
+    )
+
     Box(
         modifier = Modifier
             .padding(horizontal = pageControl.spacing / 2)
             .clip(Shape.Pill.toShape())
             .background(color)
-            .size(width = width, height = height),
+            .size(width = width, height = height)
+            .conditional(shouldApplyStroke) {
+                border(width = strokeWidth, color = strokeColor, shape = Shape.Pill.toShape())
```

Comment:
This border applies on top of the indicator itself, without changing its size. This differs from the border parameter in the stacks or other components, where the border can actually modify the size of the whole component. Not sure if that's the expected behavior... Thoughts?